tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/ConfigurationFixture.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Tests
+{
+    public class ConfigurationFixture : IDisposable
+    {
+        public ConfigurationFixture()
+        {
+            Configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: false)
+                .Build();
+        }
+
+        public IConfiguration Configuration { get; private set; }
+
+        public void Dispose()
```

Comment:
I don't think this dispose is actually doing anything.  Is this class needed?

Reply:
We removed this class. Thanks!